src/components/WhaleSightingViewer/WhaleSightingViewer.tsx

```diff
@@ -1,32 +1,31 @@
 import { useEffect, useState } from "react";
 import SightingList from "./SightingList";
-import { getWhaleSightings, NewSighting } from "../../clients/apiClient";
+import { fetchAllApprovedSightings, WhaleSighting } from "../../clients/apiClient";
 import "./WhaleSightingViewer.scss";
 
 export function WhaleSightingViewer() {
 
 	const [page, setPage] = useState(1);
-
-	const [sightingsList, setSightingsList] = useState<NewSighting[]>();
+	
+	const [sightings, setSightings] = useState<WhaleSighting[]>();
 
     useEffect(() => {
-        getWhaleSightings()
-            .then(data => setSightingsList(data));
+        fetchAllApprovedSightings()
+            .then(data => setSightings(data));
       }, []);
 
-
-	if (!sightingsList) return <p>Waiting for data...</p>
+	if (!sightings) return <p>Waiting for data...</p>
 
 	return 	<>
 		<h2 className="whale-sighting-heading">Whale Sighting Viewer</h2>
 		<div className="whale-sighting-page">
 			<div className="whale-sighting-map-view-button">Switch to Map View</div>
 			<div className="whale-sighting-filters">Filters to go here</div>
 			<div className="whale-sighting-sort">Sort to go here</div>
-			{<SightingList pageNum = {page} sightingsList = {sightingsList}/>}
+			<SightingList pageNum = {page} sightings = {sightings}/>
 			<div className="page-buttons">
 				{page > 1 ? <p className="prevlink" onClick={() => setPage(page - 1)}>Previous</p> :<></>}
```

Comment:
Can we rewrite these to make them easier to read?
```suggestion
				{page > 1 
				    ? <p className="prevlink" onClick={() => setPage(page - 1)}>Previous</p>
				    :<></>}
```